examples/synthetics_mesh_example_test.go

```diff
@@ -212,38 +267,45 @@ func printMatrixRows(matrix metricsMatrix, w *tabwriter.Writer) {
 	}
 }
 
-func formatCell(metrics *syntheticspb.MetricData) string {
-	if metrics == nil {
+func formatPingLatency(pr *syntheticspb.PingResults) string {
+	return formatMetricData(pr.GetLatency())
+}
+
+func formatPingJitter(pr *syntheticspb.PingResults) string {
+	return formatMetricData(pr.GetJitter())
+}
+
+func formatPingPacketLoss(pr *syntheticspb.PingResults) string {
+	pl := pr.GetPacketLoss()
+	if pl == nil {
+		return "[X]\t"
+	}
+
+	return fmt.Sprintf("%.2f %% / %v\t", toPercent(pl.GetCurrent()), pl.GetHealth())
+}
+
+func toPercent(v float64) float64 {
+	return v * 100
+}
+
+func formatMetricData(md *syntheticspb.MetricData) string {
+	if md == nil {
 		return "[X]\t"
 	}
 
 	return fmt.Sprintf(
 		"%v / %v / %v / %v\t",
-		formatMetricValue(metrics.GetCurrent()),
-		formatMetricValue(metrics.GetRollingAvg()),
-		formatMetricValue(metrics.GetRollingStddev()),
-		metrics.GetHealth(),
+		formatMetricValue(md.GetCurrent()),
+		formatMetricValue(md.GetRollingAvg()),
+		formatMetricValue(md.GetRollingStddev()),
+		md.GetHealth(),
 	)
 }
 
-// formatCell formats the value of metric given in nanosecond to millisecond value.
+// formatMetricValue formats the value of metric given in nanoseconds to milliseconds.
 func formatMetricValue(metricValue uint32) string {
 	if metricValue == 0 {
```

Comment:
Nit: conceivably, 0 is a valid value for jitter. It is of course unlikely, but (for example) an idle agent running on an idle server may measure very low latency variance when pinging a nearby target via mostly idle high-bandwidth network. It may even get truncated to 0.

Reply:
Good catch.
Fixed in: https://github.com/kentik/community_sdk_golang/pull/96